ethers-core/src/types/transaction/response.rs

```diff
@@ -132,7 +132,7 @@ pub struct Transaction {
     pub chain_id: Option<U256>,
 
     /// Captures unknown fields such as additional fields used by L2s
-    #[cfg(not(any(feature = "celo", feature = "optimism")))]
+    #[cfg(not(any(feature = "celo")))]
     #[serde(flatten)]
     pub other: crate::types::OtherFields,
```

Comment:
okay, I think this is reasonable to always enable this because this is also useful for other chains that include additional fields